src/css/components/_modal-dialog.scss

```diff
@@ -18,6 +18,8 @@
 .vjs-modal-dialog .vjs-modal-dialog-content {
   @extend %fill-parent;
 
+  @include user-select(auto);
```

Comment:
[mdn says](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#Syntax) it "applies to subelements", though it's not inherited. Is this something we need to worry about more broadly?